python/ray/_private/runtime_env/conda_utils.py

```diff
@@ -1,5 +1,6 @@
 import logging
```

Comment:
@architkulkarni the context for the changes to this file were to (1) log the output of the `conda create` command to the passed logger and (2) return the last N lines of the command so we can insert it into the exception on failure. I ended up splitting out the `exec_command` function because the two codepaths were completely different after my changes.